corehq/apps/domain/_design/views/docs/map.js

```diff
@@ -1,14 +0,0 @@
-function (doc) {
-    // !code util/doc_utils.js
-
-    var i,
-        date,
-        domains;
-    if (doc.doc_type) {
-        date = get_date(doc);
-        domains = get_domains(doc);
-        for (i = 0; i < domains.length; i += 1) {
-            emit([domains[i], doc.doc_type, date], null);
```

Comment:
`"domain/docs"` was replaced by  `"by_domain_doc_type_date/view"` in the PR this one is based off of: https://github.com/dimagi/commcare-hq/pull/9137/files (You can find some examples in there of how to do the switch - in most cases it's trivial).  I'll copy/paste this to the ticket you linked.